AUTHORS

```diff
@@ -1 +1,2 @@
 serge-sans-paille <sguelton@quarkslab.com>
+Jérôme Dumesnil
```

Comment:
Thanks ;-) ! You can also add my mail <jerome.dumesnil@gmail.com> if you want to.

Reply:
done (and for @cbeck88 too btw)